docs/developer-guide/install.md

```diff
@@ -0,0 +1,74 @@
+---
+order: 2
+description: Steps to install Starport on your local computer.
+---
+
```

Comment:
let's look at https://tutorials.cosmos.network/starport/ and see what we can use/reuse